modules/ask_va_api/app/lib/ask_va_api/inquiries/payload_builder/shared_helpers.rb

```diff
@@ -5,26 +5,32 @@ module Inquiries
     module PayloadBuilder
       module SharedHelpers
         def fetch_country(country_code)
+          return if country_code.nil?
+
           country_code = 'US' if country_code == 'USA'
           I18n.t("ask_va_api.countries.#{country_code}", default: country_code)
         end
 
         def fetch_state(state_code)
+          return if state_code.nil?
+
           I18n.t("ask_va_api.states.#{state_code}", default: state_code)
         end
 
         def formatted_pronouns(pronouns)
-          return pronouns[:pronouns_not_listed_text] if pronouns[:pronouns_not_listed_text].present?
+          return unless pronouns
 
-          pronouns&.key(true).to_s.tr('_', '/')
+          pronouns[:pronouns_not_listed_text].presence || pronouns.key(true)&.to_s&.tr('_', '/')
         end
 
         def contact_field(field, inquiry_details, inquiry_params)
           inquiry_details[:level_of_authentication] == 'Business' ? inquiry_params[field] : nil
         end
 
         def fetch_state_code(state)
-          I18n.t('ask_va_api.states').invert[state].to_s
+          return if state.nil?
```

Comment:
Are there any tests for all of the `nil` returns happening, including this one?

Reply:
Yes, it's in the shared_context.rb

![Screenshot 2024-10-28 at 3 25 14 PM](https://github.com/user-attachments/assets/2430dffd-f60e-46b2-a270-b1ebb740762a)

Reply:
ok, thanks.